app/components/footerlinks/footerlinks.js

```diff
@@ -77,13 +78,27 @@ const FooterLinks = (props) => {
           onClick={metricFnMkr('Support')}
           target="_blank">{t(SupportText)}</a>
     </div>
-    <div className='footer-link'>       
+    {__BRANDING__ === 'diabeloop' ? (
+       <div className='footer-link'>       
+        <a
+          href={urlPrivacy}
+          id='privacy'
+          onClick={metricFnMkr('Privacy')}
+          target='_blank'>{t('Privacy Policy')}</a> 
+        &nbsp;&&nbsp;
```

Comment:
```suggestion
```
React ignore this

Reply:
Yep it's ignored by React but work as expected